src/domain/entities/testing/mocks.ts

```diff
@@ -259,8 +259,12 @@ export const generateFigmaTeamSummary = ({
 	teamId = uuidv4(),
 	teamName = uuidv4(),
 	authStatus: status = FigmaTeamAuthStatus.OK,
+	webhookId = uuidv4(),
+	webhookPasscode = uuidv4(),
 }: Partial<FigmaTeamSummary> = {}): FigmaTeamSummary => ({
 	teamId,
 	teamName,
 	authStatus: status,
+	webhookId,
+	webhookPasscode,
```

Comment:
**issue:** It can be safer to keep `webhookPasscode` internal to the application and do not expose it to outside via API. Currently, the frontend is not aware (and should not be aware) of `webhookId` and `webhookPasscode` -- these fields are considered as internal implementation details.

The intention of the change that broke the build was to return `FigmaTeamSummary` (with `teamId`, `teamName` and `authStatus`) from the `/admin/teams/:teamId/connect` endpoint for the frontend convenience. 

We discussed this with Roy and addressed this within this PR!129:
https://github.com/atlassian-labs/figma-for-jira/pull/129/files#diff-473697600a010a6f00bb27d2fa9a0c1d0bcd7bfa25071d3b203318cb8c6d7c2fR52

PR!129 also fixes the build, so I believe we could cancel this PR. I apologise for the miscommunication, @elainefigma .

Reply:
Sounds good - thanks for the context!